src/openfl/display/DisplayObject.hx

```diff
@@ -1343,6 +1343,17 @@ class DisplayObject extends EventDispatcher implements IBitmapDrawable #if openf
 		
 	}
 	
+	private static inline function __calculateAbsoluteTransform (local:Matrix, parentTransform:Matrix, target:Matrix):Void {
```

Comment:
Methods in openfl are alphabetically ordered. Please place the method in a proper place.

Reply:
That's one of way of organizing code...